modules/plugin/coverage-multidim/netcdf/src/main/resources/org/geotools/imageio/netcdf/netcdf-unit-replacements.properties

```diff
@@ -2,4 +2,7 @@ microgrammes\ per\ cubic\ meter=µg*m^-3
 DU=µmol*m^-2*446.2
 m2=m^2
 m3=m^3
-s2=s^2
\ No newline at end of file
+s2=s^2
```

Comment:
This change is going to break, on upgrade, the existing installations using the units replacements customization (visible not only to GeoTools users, but GeoServer users alike, which are often not developers)

What kind of changes are expected and what can be done to fix them? Is it possible to write a guide, and/or is there a table listing before/after differences?

Reply:
Not sure how the units library handled these units in the netcdf format before but adding these ensured that it could parse these as the appropriate temperature units.  I am not familiar with the netcdf format so someone else would have to address updating the documentation.